### PR TITLE
temporarily switch b64 sig to hex

### PR DESCRIPTION
**Pull request by heartsucker**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,6 +38,9 @@ help:
 new: image ## Generate a new config then run the client.
 	$(DOCKER_RUN) --net=host $(IMAGE_SOTA)
 
+build: ## Build the binary in debug mode
```
Comment (taheris) [inline Makefile]: client-dev?

Comment (heartsucker) [inline Makefile]: Fixed

Comment (heartsucker) [inline Makefile]: Fixed

Comment (taheris) [inline Makefile]: I meant there was already a `client-dev` section :)

**Pull request merged**